extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/musicModels/GooglePlaylist.java

```diff
@@ -29,11 +32,14 @@ public class GooglePlaylist {
   @JsonProperty("description")
   private String description;
 
+  // In JSON format, the Timestamp type is encoded as a string in the
+  // [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format. That is, the
+  // format is "{year}-{month}-{day}T{hour}:{min}:{sec}[.{frac_sec}]Z".
   @JsonProperty("createTime")
-  private long createTime;
+  private String createTime;
 
   @JsonProperty("updateTime")
```

Comment:
```suggestion
  /** Time the playlist was last modified in any way. Format identical to {@link createTime}'s. */
  @JsonProperty("updateTime")
```

Reply:
Done in latest commit 7b70e28051bdc5a540f2c97e16d53578cb5601fd.